### ISSUE TITLE
Localization improvement request

### ISSUE THREAD

**Issue by Ghostlyr** [huanghongxun/WaterPower#21]
Hello. Thanks for wonderful mod but there is the problem with its correct localization in some languages, for example - Russian. Because of the accession of the localized names, endings of words are obtained incorrect. I ask to make a individually localized names for all blocks and items. Let it be longer occupy space in the file, but the translation will be correct.

**Comment by huanghongxun** [comment]
What about "zinc plate"? Цинк Пластина or Пластина Цинк?

**Comment by Ghostlyr** [comment]
Both variants are incorrect, this must be translated as "Цинковая пластина".

**Comment by huanghongxun** [comment]
"Цинковая" is an adjective? Can I solve the problem by having a list for adjectives specially?

**Comment by Ghostlyr** [comment]
Yes. I can make a list of adjectives, but still it would be better if localized names of each items has been individual, because in some occasion need to to change the word order... :(

**Comment by huanghongxun** [comment]
For example? which item/block?
Maybe there is a better way: cptwtrml.reservoir.RESERVOIR="{type} Reservoir Block" or "{type_adj} Reservoir Block" or "Reservoir {type}"
And my mod will replace "{type}" or "{type_adj}" to cptwtrml.reservoir.WOOD or cptwtrml.reservoir.WOOD.adj.
So you can change the order or the word form.

**Comment by 3TUSK** [comment]
@huanghongxun You probably want to look at this: https://github.com/ganymedes01/Another-One-Bites-the-Dust/issues/32#issuecomment-153629944
